spec/features/user_creates_a_dinosaur_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+feature 'user can create a new dinosaur', %{
+  As a user
+  I want to create a new dinosaur
+  So that I can share my favorite dinosaurs
+
+  Acceptance Criteria
+  [√] User can navigate to new dinosaur form from index
+  [√] User can view form properly
+  [√] User gets confirmation if dinosaur is entered correctly
+  [√] User gets errors if form is filled out incorrectly
+  [√] User gets redirected to show page of created dinosaur
+
+} do
+
+  scenario "User can navigate to form from index" do
+
+    visit root_path
+    click_link "Create a dinosaur"
+    expect(page).to have_content("Create a Dinosaur!")
+
+  end
+
+  scenario "Form fields are visible" do
+
+    visit new_dinosaur_path
+    expect(page).to have_content("Name")
+    expect(page).to have_content("Location found")
+    expect(page).to have_content("Info url")
```

Comment:
I'd favor looking specifically for the field here, using `have_field` in lieu of `have_content`.